nmdc_server/auth.py

```diff
@@ -250,7 +251,35 @@ async def orcid_authorize(request: Request, db: Session = Depends(get_db)):
     # TODO: use token_response.access_token to make requests to the ORCID API for more user info?
     # Or possibly store the access token in the database for later use?
     user = User(orcid=token_response["orcid"], name=token_response["name"])
+
+    orcid_email_response = await oauth2_client.orcid.get(
+        "https://pub.orcid.org/v3.0/0009-0004-9185-506X/email", token=token_response
+    )
```

Comment:
```suggestion
    orcid_email_response = await oauth2_client.orcid.get(
        f"https://pub.orcid.org/v3.0/{token_response['orcid']}/email", token=token_response
    )
```

I think this URL should use the current user's ORCID

Reply:
Ugh, can't believe I forgot to change that.

Reply:
removed in 3c6ce981d57ae696db0df296997f1ab1e62521ed